test/services/notificationHubsManagement/registration-tests.js

```diff
@@ -0,0 +1,212 @@
+﻿//
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+var should = require('should');
+var azure = require('azure');
```

Comment:
`var azure = require('../../../lib/services/serviceBus/lib/servicebus');`

Reply:
will make the change